### PR TITLE
Move Wordpress root to /web, as per original build.

### PR DESCRIPTION
**Pull request by xtfer**
This moves the Wordpress root to /web (not /web/wp) as per the original example. This will fix the issue with wp-content directory paths being incorrect, and is a better solution than routing those paths into web/wp, as it allows local development to occur without rebuilds on every code change.

### CODE REVIEW

File: templates/wordpress/files/composer.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (Crell) [inline templates/wordpress/files/composer.json]: This file comes from upstream so we shouldn't be overwriting it.  Modifying via script only.

File: templates/wordpress/files/web/index.php [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?php
```
Comment (Crell) [inline templates/wordpress/files/web/index.php]: Does this come from upstream, or is it vanilla ours?

File: dodo.py [hunk 3]
```diff
@@ -12,7 +12,6 @@
 from project.laravel import Laravel
 from project.magento import Magento2ce
 from project.symfony import Symfony3, Symfony4
-from project.wordpress import Wordpress
```
Comment (Crell) [inline dodo.py]: It's not going to work without the class available, methinks...

File: templates/wordpress/files/composer.json (hunk 1 above)
Comment (xtfer) [reply templates/wordpress/files/composer.json]: I don't understand why we do this. Using one composer file to pull another dependency with a composer file is design intent of Composer. In this case, we set the relevant install paths for the upstream in our file. This is safer than overwriting the source, IMHO.

File: templates/wordpress/files/web/index.php (hunk 2 above)
Comment (xtfer) [reply templates/wordpress/files/web/index.php]: It's ours. Moving the wp web root doesn't work without a custom index.php

**Pull request closed without merging**